scripts/exregional_run_gsidiag.sh

```diff
@@ -189,8 +189,14 @@ else
   analworkname="_gsi"
 fi
 
-analworkdir_conv="${cycle_dir}/anal_conv${analworkname}"
-analworkdir_dbz="${cycle_dir}/anal_radardbz${analworkname}"
+analworkdir_conv_dbz="${cycle_dir}/anal_conv_dbz${analworkname}"
```

Comment:
Can these move to jobs/JREGIONAL_RUN_GSIDIAG to be more consistent with other tasks?

Reply:
This is not that easy to be done in J-job.